### PR TITLE
Feature: Implement the logic to get InPage recommendation texts

### PR DESCRIPTION
**Pull request by akueisara**
- [x] Create VirtusizeView as the superclass to let custom Virtusize views to be passed to function Virtusize.setupVirtusizeView, previously named as setupVirtusizeButton.
- [x] Build the function getRecommendationText in StoreProduct.kt
- [x] Add localization texts for English, Japanese and Korean

### CODE REVIEW

File: libsource/src/main/java/com/virtusize/libsource/data/parsers/UserDataJsonParser.kt [hunk 1]
```diff
@@ -18,9 +18,9 @@ internal class UserDataJsonParser: VirtusizeJsonParser {
 
     companion object {
```
Comment (zr0z) [inline libsource/src/main/java/com/virtusize/libsource/data/parsers/UserDataJsonParser.kt:19]: Those can be safely discarded, they are part of the legacy Purchase history flow and are not used in our modern products

File: libsource/src/main/res/values-ja/strings.xml [hunk 2]
```diff
@@ -2,4 +2,15 @@
 <resources>
```
Comment (zr0z) [inline libsource/src/main/res/values-ja/strings.xml:2]: Those should come from [i18n api endpoint](https://i18n.virtusize.jp/bundle-payloads/aoyama/ja) and not be static in the app.

Each languages:

- https://i18n.virtusize.jp/bundle-payloads/aoyama/ja
- https://i18n.virtusize.jp/bundle-payloads/aoyama/ko
- https://i18n.virtusize.jp/bundle-payloads/aoyama/en

File: libsource/src/main/res/values-ko-rKR/strings.xml [hunk 3]
```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="utf-8"?>
```
Comment (zr0z) [inline libsource/src/main/res/values-ko-rKR/strings.xml:1]: Same as above

File: libsource/src/main/res/values/strings.xml [hunk 4]
```diff
@@ -3,5 +3,15 @@
     <string name="landscape" translatable="false">landscape</string>
```
Comment (zr0z) [inline libsource/src/main/res/values/strings.xml:3]: Same as above

File: libsource/src/test/java/com/virtusize/libsource/TestFixtures.kt [hunk 5]
```diff
@@ -31,10 +32,10 @@ internal object TestFixtures {
         """
```
Comment (zr0z) [inline libsource/src/test/java/com/virtusize/libsource/TestFixtures.kt:32]: As I said we shouldn't parse those values

File: libsource/src/main/java/com/virtusize/libsource/data/parsers/UserDataJsonParser.kt (hunk 1 above)
Comment (akueisara) [reply libsource/src/main/java/com/virtusize/libsource/data/parsers/UserDataJsonParser.kt]: I removed the UserData class and only parsed should_see_ph_tooltip in the Data class. 
See the commit 27f90a3

File: libsource/src/main/res/values-ja/strings.xml (hunk 2 above)
Comment (akueisara) [reply libsource/src/main/res/values-ja/strings.xml:2]: OK, I just created the ticket for it. will work on it and create another PR

File: libsource/src/test/java/com/virtusize/libsource/TestFixtures.kt (hunk 5 above)
Comment (akueisara) [reply libsource/src/test/java/com/virtusize/libsource/TestFixtures.kt]: I removed those test fixtures. See the commit 91c01dd

**Pull request merged**